src/app/test/api/http/unit/handlers/v1/command_publishing_blocklist_test.py

```diff
@@ -0,0 +1,394 @@
+    yield issue_token_pair(user)["access"]
+
+
+@pytest.fixture
+def blocklist_cleanup():
+    yield
+    CommandPublishingBlockList.drop_collection()
+
+
+class TestCommandPublishingBlockListAPI:
+    @pytest.mark.gen_test
+    def test_delete(self, http_client, base_url, blocklist):
+        url = f"{base_url}/api/v1/commandpublishingblocklist/{blocklist.id}"
+        request = HTTPRequest(url, method="DELETE")
+
+        response = yield http_client.fetch(request)
+
+        assert response.code == 204
+        assert len(CommandPublishingBlockList.objects.filter(id=blocklist.id)) == 0
+
+    @pytest.mark.gen_test
+    def test_delete_404(self, http_client, base_url):
+        url = f"{base_url}/api/v1/commandpublishingblocklist/624d8e8cce329715faf172d5"
+        request = HTTPRequest(url, method="DELETE")
+
+        with pytest.raises(HTTPError) as excinfo:
+            yield http_client.fetch(request)
+
+        assert excinfo.value.code == 404
+
+    @pytest.mark.gen_test
+    def test_delete_auth(
+        self,
+        http_client,
+        base_url,
+        blocklist,
+        app_config_auth_enabled,
+        access_token_user_admin,
+    ):
+        url = f"{base_url}/api/v1/commandpublishingblocklist/{blocklist.id}"
+        headers = {
+            "Authorization": f"Bearer {access_token_user_admin}",
+            "Content-Type": "application/json",
+        }
+        request = HTTPRequest(url, headers=headers, method="DELETE")
+
+        response = yield http_client.fetch(request)
+
+        assert response.code == 204
+        assert len(CommandPublishingBlockList.objects.filter(id=blocklist.id)) == 0
+
+    @pytest.mark.gen_test
+    def test_delete_auth_403_user_no_permisions(
+        self,
+        http_client,
+        base_url,
+        blocklist,
+        app_config_auth_enabled,
+        access_token_user,
+    ):
+        url = f"{base_url}/api/v1/commandpublishingblocklist/{blocklist.id}"
+        headers = {
+            "Authorization": f"Bearer {access_token_user}",
+            "Content-Type": "application/json",
+        }
+        request = HTTPRequest(url, headers=headers, method="DELETE")
+
+        with pytest.raises(HTTPError) as excinfo:
+            yield http_client.fetch(request)
+
+        assert excinfo.value.code == 403
+
+    @pytest.mark.gen_test
+    def test_delete_auth_401_no_token(
+        self, http_client, base_url, blocklist, app_config_auth_enabled
+    ):
+        url = f"{base_url}/api/v1/commandpublishingblocklist/{blocklist.id}"
+        request = HTTPRequest(url, method="DELETE")
+
+        with pytest.raises(HTTPError) as excinfo:
+            yield http_client.fetch(request)
+
+        assert excinfo.value.code == 401
+
+    @pytest.mark.gen_test
+    def test_get(self, http_client, base_url, blocklist):
+        url = f"{base_url}/api/v1/commandpublishingblocklist/"
+
+        request = HTTPRequest(url, method="GET")
+
+        response = yield http_client.fetch(request)
```

Comment:
Mostly just an FYI, but you don't need to construct a HTTPRequest for a basic "GET".  That's the default behavior if you just pass in a URL here, so if you cared to you could just do:

```python
response = yield http_client.fetch(url)
```